MDANSE/Src/MDANSE/Framework/Jobs/NeutronDynamicTotalStructureFactor.py

```diff
@@ -416,12 +417,12 @@ def finalize(self):
             ][:]
 
         # Compute incoherent functions and structure factor
-        for element in nAtomsPerElement:
+        for element, number in list(nAtomsPerElement.items()):
```

Comment:
No need to cast to list.
```suggestion
        for element, number in nAtomsPerElement.items():
```